tests/benchmarks/multicore/idle_pwm_led/Kconfig.sysbuild

```diff
@@ -7,4 +7,5 @@
 source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
 
 config REMOTE_BOARD
-	string "The board used for remote target"
+	string
+	default "$(BOARD)/$(SOC)/cpurad" if SOC_NRF54H20_CPUAPP
```

Comment:
@tejlmand : Or shall I use
default "$(BOARD)/**nrf54h20**/cpurad" if SOC_NRF54H20_CPUAPP
as in https://github.com/nrfconnect/sdk-nrf/pull/19018#discussion_r1853704191?

Reply:
the `$(SOC)` is not defined as a variable you can use in Kconfig, so the actual expansion of this line becomes:
`nrf54h20dk//cpurad`.

So at least you should remove `$(SOC)` as it gives the wrong impression that this is a valid var.

The reason that `nrf54h20dk//cpurad` is also a valid board target when the board is single SoC.

Reply:
Changed to `default "$(BOARD)/nrf54h20/cpurad" if SOC_NRF54H20_CPUAPP`